README.md

```diff
@@ -1,45 +1,76 @@
 # Sentry Release GitHub Action
-Automatically create a Sentry Release in a workflow.
-
-## Why Releases?
-A release is a version of your code that is deployed to an environment. 
-When you give Sentry information about your releases, you unlock a number of new features:
- - Determine the issues and regressions introduced in a new release
- - Predict which commit caused an issue and who is likely responsible
- - Resolve issues by including the issue number in your commit message
- - Receive email notifications when your code gets deployed
-
-Additionally, releases are used for applying source maps to minified JavaScript to view original, untransformed source code.
+Automatically create a Sentry release in a workflow. 
+You can learn more about releases in [releases documentation](https://docs.sentry.io/workflow/releases).
 
 ## Usage
 Adding the following to your workflow will create a new Sentry release for the 
-`production` environment of your project and upload JavaScript source maps from
-the `./lib` directory. 
+`production` environment of your project with an automatically generated `version` name.
   
 ```yaml
-uses: getsentry/action-release@v1
-with:
-  environment: 'production'
-  sourcemaps: './lib'
+jobs:
+  release:
+    runs-on: ubuntu-latest
```

Comment:
Do you think it makes sense to have the outer definition here?

Reply:
I wanted to capture the environment variables, but it'll be a lot simpler if I pass them into the step.